src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1846,5 +1846,32 @@ public static boolean isXnp(final String reference, final String alternate) {
 
         return result;
     }
+
+    /**
+     * Returns true if the context represents an MNP. If the context supplied contains the GVCF NON_REF symbolic
+     * allele, then the determination is performed after discarding the NON_REF symbolic allele.
+     *
+     * @param ctx a Variant context
+     * @return true if the context represents an unmixed MNP (i.e. all alleles are non-symbolic and of the same
+     *         length > 1), false otherwise
+     */
+    public static boolean isMnpWithoutNonRef(final VariantContext ctx) {
```

Comment:
This method name is a little confusing...maybe `isUnmixedMnpIgnoringNonRef()` would be a (slightly) better/more precise name?